.gitignore

```diff
@@ -4,4 +4,3 @@ node_modules
 eslintignore
 jest.config.js
 tsconfig.json.save
-DS_Store
```

Comment:
You should add .DS_Store to gitignore too. And somehow I think it's still committed to the branch.